src/System.Console/tests/Color.cs

```diff
@@ -92,6 +92,68 @@ public static void NonRedirectedOutputDoesUseAnsiSequences()
         Assert.Equal(3, Encoding.UTF8.GetString(data.ToArray()).ToCharArray().Count(c => c == Esc));
     }
 
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        foreach (string location in ConsolePal.TermInfo.Database._terminfoLocations)
+        {
+            // Skip directories that dont exist
+            if (Directory.Exists(location) == false)
+                continue;
+
+            foreach (string term in Directory.EnumerateFiles(location, "*", SearchOption.AllDirectories))
+            {
+                if (term.ToUpper().Contains("README")) continue;
+                ConsolePal.TermInfo.Database db = ConsolePal.TermInfo.Database.ReadDatabase(Path.GetFileName(term));
+                Assert.True(db != null, string.Format("Could not parse terminfo file {0}", term)); // Don't use Assert.NotNull so we can print out the term info file on failure
```

Comment:
The value of the [Theory] approach is that a failed assert for one input doesn't prevent subsequent inputs from being tested, since each is logically considered its own test case.  It also has the benefit of keeping the generation of the inputs separate from the code actually doing the testing/verification. But it's not a big deal.